app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -109,23 +109,14 @@ def update
     param :major, :number, :desc => N_("Override the major version number"), :required => false
     param :minor, :number, :desc => N_("Override the minor version number"), :required => false
     param :environment_ids, Array, :desc => N_("Identifiers for Lifecycle Environment"), :required => false
-    param :is_force_promote, :bool, :desc => N_("force content view promotion and bypass lifecycle environment restriction"), :required => false
+    param :check_needs_publish, :bool, :desc => N_("Check if content view needs a publish even based on audited changes since last publish"), :required => false
```

Comment:
Yeah, I'd go even farther

```suggestion
    param :publish_only_if_needed, :bool, :desc => N_("Check audited changes and proceed only if content or filters have changed since last publish "), :required => false
```